manager/integration/tests/test_settings.py

```diff
@@ -653,11 +656,78 @@ def test_setting_backing_image_auto_cleanup():  # NOQA
         2. The related backing image file is removed.
         3. The download state of other disks keep unchanged.
            All volumes still work fine.
-    8. Delete all volumes. Verify that all states in the backing image disk map
-       will become terminating first,
-       and all entries will be removed from the map later.
+    8. Delete all volumes. Verify that there will only remain 1 entry in the
+       backing image disk map
```

Comment:
Why still remaining 1 entry if all volumes deleted and BIM auto cleanup executed?

Reply:
Hi, @innobead ,

I checked with @shuo-wu, by default there will be 1 file in backing image data map to avoid download again if user want to re use the backing image, so if we delete all volumes which using that backing image, there should be still 1

Reply:
Yeah, thanks. Keeping 1 is for the downloading case.